packages/messages/src/browser/notifications.ts

```diff
@@ -72,8 +74,19 @@ export class Notifications {
                 button.addEventListener('click', () => {
                     action.fn(handler);
                     close();
+                    clearTimeout(this.closeTimer);
                 });
             }
+            let adjustedCloseTimer = this.defaultAutoCloseTimeout;
+            // Give more time before activating the autoClose Notification when user has choices
+            if (properties.actions !== undefined && properties.actions.length > 1) {
+                adjustedCloseTimer = adjustedCloseTimer * properties.actions.length;
```

Comment:
Just for reference, we discussed this offline and the conclusion is that it's better to make it auto-dismissable for certain severity level i.e info/warning and make it must-be-dismissed-manually for things like errors. 

Also I don't think using the number of possible actions in the message is a good factor to use for the duration of the message, as there's probably no correlation between the number of actions a user can do on that message and the severity of it.

Reply:
I overlooked your comment, as already collapsed.

That's the thing, all kind of use cases require different options. I think, it's up to the `MessageService` clients to decide about timeout and other properties of messages. OTOH we can have default options for most common cases.